### ISSUE TITLE
拖动报错，麻烦看看

### ISSUE THREAD

**Issue by nowui** [harrel56/use-sortablejs#219]
demo地址：[https://codesandbox.io/p/sandbox/hardcore-panka-w4t5d9?file=%2Fsrc%2Findex.js%3A10%2C30](url)

拖动任意项后会报错，报错看截图
![image](https://github.com/user-attachments/assets/4af5de67-bf2e-4d45-a481-258e17f7deff)

**Comment by harrel56** [comment]
Well, it looks like I was wrong to believe jsdoc and typescript telling me about field being not-nullable :D As a quick workaround you can add this to your code:
```js
import Sortable, {Swap} from 'sortablejs';

Sortable.mount(new MultiDrag());
```

I'm gonna release fix for this soon - thanks for reporting it!

**Comment by harrel56** [comment]
Published the fix with `1.1.7` version - feel free to reopen this issue if anything is still not right

**Comment by nowui** [comment]
It has been fixed, thank you
